src/mapping/nav2_map_server/test/integration/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+include_directories( ${PROJECT_SOURCE_DIR}/test )
+
+ament_add_gtest_executable(test_occ_grid_node
+  test_occ_grid_node.cpp
+  ${PROJECT_SOURCE_DIR}/test/test_constants.cpp
+)
+ament_target_dependencies(test_occ_grid_node rclcpp nav_msgs)
+target_link_libraries(test_occ_grid_node
+  map_reps
+  ${ament_LIBRARIES}
+)
+
+ament_add_test(test_occ_grid_cpp
+  GENERATE_RESULT_FOR_RETURN_CODE_ZERO
+  COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/test_occ_grid_launch.py"
+  WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
+  ENV
+    TEST_LAUNCH_DIR=${TEST_LAUNCH_DIR}
```

Comment:
Two space indentation.

Reply:
In this case, the variable is part of a list of ENV variables, so I figured it was best to indent it from ENV to show a 'sort of' scoping.